packages/variable/package.json

```diff
@@ -32,6 +32,8 @@
     "eslint-plugin-react": "^7.14.3",
     "jest": "^24.8.0",
     "jsinspect": "^0.12.7",
-    "prettier": "^1.18.2"
+    "prettier": "^1.18.2",
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.3.10"
```

Comment:
- [ ] these can be reverted
- [ ] actually, I _believe_ all changes to the `variable` sub-package can be reverted! (except for this line:)
  https://github.com/readmeio/api-explorer/blob/6a2031c8253b00f093a814d0fa6be97f1304cae7/packages/variable/index.jsx#L170